api/src/Controller/ManageController.php

```diff
@@ -62,7 +62,7 @@ private function dbInfo()
             if ($e instanceof \PDOException && $e->getCode() === 7) {
                 $returnMessage = 'Database service not reachabe (' . $e->getMessage() . ')';
             }
-            if ($e instanceof \Doctrine\DBAL\DBALException) {
+            if ($e instanceof \Exception) {
```

Comment:
I wouldn't necessarily say an `Exception` type signifies the migrations table is missing...if there's no alternative to `Doctrine\DBAL\DBALException` we may as well lose the if block completely and leave the generic error message defined above.